kview/src/main/java/dev/ikm/komet/kview/mvvm/view/pattern/PatternDetailsController.java

```diff
@@ -314,8 +292,40 @@ public void onChanged(Change<? extends DescrName> change) {
         fqnAddDateLabel.textProperty().bind(dateStrProp);
         // Setup Properties
         setupProperties();
+        setupContextMenuPopUp();
     }
 
+    private void setupContextMenuPopUp() {
```

Comment:
Can this method be more specific?

Reply:
Not sure what you mean. I have changed the name to setupContextMenuMenuItems